src/VirtualClient/VirtualClient.Main/profiles/PERF-STRESSNG.json

```diff
@@ -4,15 +4,18 @@
     "Metadata": {
         "RecommendedMinimumExecutionTime": "00:05:00",
         "SupportedPlatforms": "linux-x64,linux-arm64",
-        "SupportedOperatingSystems": "CBL-Mariner,CentOS,Debian,RedHat,Suse,Ubuntu"
+        "SupportedOperatingSystems": "CBL-Mariner,CentOS,Debian,RedHat,Suse,Ubuntu",
+        "SpecialInstrucions":  "The commandline parameter --yaml is programatically added, do not add it in profile. By default --cpu is set to ProcessorCount."
```

Comment:
We can add information about default timeout , cpu threads and metrics flag is on.

Reply:
Modified the Special Instructions